device-base/generate-dockerfile.sh

```diff
@@ -18,6 +18,7 @@ bb_rcn_ee_key_cmd='wget http://repos.rcn-ee.net/debian/pool/main/r/rcn-ee-archiv
 devices='raspberrypi2 beaglebone edison'
 suites='jessie wheezy'
 
+chmod +x entry-edison-nosystemd.sh
```

Comment:
You can add the executable flag in the repo itself, no need to do it in a script every time.